tests/Fixer/ClassNotation/OrderedTypesFixerTest.php

```diff
@@ -543,28 +517,10 @@ protected function bar(string $str, ?array $config = null): callable {}
     return $e;
 }
 ',
+            null,
+            ['sort_algorithm' => 'alpha', 'null_adjustment' => 'none'],
         ];
-    }
-
-    /**
-     * @dataProvider provideFixWithSandwichedWhitespaceOrCommentInTypeCases
-     *
-     * @requires PHP 8.0
-     */
-    public function testFixWithSandwichedWhitespaceOrCommentInType(string $expected, ?string $input = null): void
-    {
-        // The current design of the fixer uses the `TypeAnalysis` class which collects the
-        // types and ignores whitespaces and comments. These ignored tokens are not accounted
-        // during instantiation of the TypeAnalysis class, thus we have no way of recovering
-        // those token information. This test case mainly proves this fact.
```

Comment:
what happen with this part?

Reply:
Tests are kept, they are checking extra whitespace around tokens, and we test such cases in many places, I'm not sure how the comment is helpful here.